### PR TITLE
Add new blogpost to lacchain eosio annoucement

### PR DESCRIPTION
**Pull request by AngeloCG97**

### CODE REVIEW

File: blog/2021-07-29-lacchain-eosio-blogpost.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+tags: [eosio, blockchain, blockchain technology, lacchain, open-source]
+metaTitle: "EOS Costa Rica: What LACChain EOSIO Will Represent for Latin America"
+metaDescription: Announcement of LACChain EOSIO, a new EOSIO blockchain network in Latin America and the Caribbean supported by LACChain and IDB Lab.
+---
+
+![Get to Know LACChain EOSIO](/img/blog/lacchain-eosio-blogpost/lacchain-eosio-blogpost.jpg)
+
+## Get to Know LACChain EOSIO
+
+Today, it was announced the launch of the first testnet and the website of [LACChain EOSIO](https://eosio.lacchain.net/), a significant step in the process of creating a regional EOSIO mainnet and the second multi-purpose public-permissioned network of LACChain. LACChain EOSIO is the result of a [partnership with EOS Argentina and LACChain](https://www.coindesk.com/ethereum-and-eosio-square-up-over-enterprise-blockchain-business-in-latin-america), an initiative led by IDB Lab (Innovation Laboratory of the Inter-American Development Bank Group), to work on an EOSIO-powered blockchain network in Latin America and the Caribbean.
```
Comment (xavier506) [inline blog/2021-07-29-lacchain-eosio-blogpost.md:18]: @ldrojas  I think this sounds better  : 

Today,  the first testnet and website launch of [LACChain EOSIO](https://eosio.lacchain.net/) was announced,

Comment (xavier506) [reply blog/2021-07-29-lacchain-eosio-blogpost.md:18]: is there a date on the article? when we say today , when is today?

File: blog/2021-07-29-lacchain-eosio-blogpost.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## What to Find in LACChain EOSIO?
+
+The LACChain EOSIO network has been submitted with many simultaneous transactions to gauge its capacity and performance with positive results on the stress tests, reaching 2,157 transactions per second. Marco Allende, Tech Lead of [LACChain](https://www.lacchain.net/), stated that “with more than 100 nodes and 50 projects in the LACChain platform, EOSIO introduces new tools and features to LACChain’s users.”
+
+Among the features of LACChain EOSIO, a user will find a block explorer, a network dashboard (part of [EOSIO Dashboard](https://eosio.online/), announced last week), and technical documentation and resources to learn about the network and EOSIO blockchain technology. Further along the road, the team intends to work on an EOSIO MainNet. Developers and organizations will host and deploy system contracts based on the Proof-of-Authority (POA) consensus mechanism.
```
Comment (xavier506) [inline blog/2021-07-29-lacchain-eosio-blogpost.md:30]: Dashboard link: https://lacchain.eosio.online/

File: blog/2021-07-29-lacchain-eosio-blogpost.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+<!--truncate-->
+
+As of today, there are currently three projects deploying the LACChain EOSIO network testnet: [Lifebank](http://lifebank.io/), an open-source app to promote blood donations; [InmuTrust](https://inmutrust.com/), a blockchain-based software to improve transparency and traceability of vaccines; and [SMARTGATE](https://smartgate.tech/), a platform for in-out gate activity for the container industry. The three projects are from EOS Costa Rica, but it is now open to welcome projects from other developers globally. 
+
+## What to Find in LACChain EOSIO?
+
+The LACChain EOSIO network has been submitted with many simultaneous transactions to gauge its capacity and performance with positive results on the stress tests, reaching 2,157 transactions per second. Marco Allende, Tech Lead of [LACChain](https://www.lacchain.net/), stated that “with more than 100 nodes and 50 projects in the LACChain platform, EOSIO introduces new tools and features to LACChain’s users.”
```
Comment (xavier506) [inline blog/2021-07-29-lacchain-eosio-blogpost.md:28]: should we link to block explorer (https://eosio-explorer.lacchain.net)

File: blog/2021-07-29-lacchain-eosio-blogpost.md (hunk 1 above)
Comment (ldrojas) [reply blog/2021-07-29-lacchain-eosio-blogpost.md:18]: I agree. @AngeloCG97 please, let's change the first sentence: Today, the first testnet and website launch of LACChain EOSIO was announced, a significant...

File: blog/2021-07-29-lacchain-eosio-blogpost.md (hunk 3 above)
Comment (ldrojas) [reply blog/2021-07-29-lacchain-eosio-blogpost.md:28]: Absolutely

File: blog/2021-07-29-lacchain-eosio-blogpost.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+tags: [eosio, blockchain, blockchain technology, lacchain, open-source]
+metaTitle: "EOS Costa Rica: What LACChain EOSIO Will Represent for Latin America"
+metaDescription: Announcement of LACChain EOSIO, a new EOSIO blockchain network in Latin America and the Caribbean supported by LACChain and IDB Lab.
+---
+
+![Get to Know LACChain EOSIO](/img/blog/lacchain-eosio-blogpost/lacchain-eosio-blogpost.jpg)
+
+## Get to Know LACChain EOSIO
+
+TToday, the first testnet and website launch of [LACChain EOSIO](https://eosio.lacchain.net/), was announced, a significant step in the process of creating a regional EOSIO mainnet and the second multi-purpose public-permissioned network of LACChain. LACChain EOSIO is the result of a [partnership with EOS Argentina and LACChain](https://www.coindesk.com/ethereum-and-eosio-square-up-over-enterprise-blockchain-business-in-latin-america), an initiative led by IDB Lab (Innovation Laboratory of the Inter-American Development Bank Group), to work on an EOSIO-powered blockchain network in Latin America and the Caribbean.
```
Comment (xavier506) [inline blog/2021-07-29-lacchain-eosio-blogpost.md:18]: double T

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**